Civi/Share/CiviMRF/Connector.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Civi\Share\CiviMRF;
+
+use Civi\Api4\ShareNodePeering;
+
+class Connector {
+
+  private int $shareNodePeeringId;
+
+  private Profile $profile;
+
+  public function __construct(int $shareNodePeeringId, Profile $profile) {
+    $this->shareNodePeeringId = $shareNodePeeringId;
+    $this->profile = $profile;
+  }
+
+  public static function load(string $shareNodePeeringId) {
+    return self::loadMultiple([$shareNodePeeringId])[$shareNodePeeringId];
+  }
+
+  public static function loadMultiple(array $shareNodePeeringIds = NULL) {
```

Comment:
Return type hint. In that case additionally a PHP doc return type hint like `array<int, self> Connectors indexed by ID` would make sense.